src/Campaigns/resources/admin/components/CampaignDetailsPage/Tabs/Settings.tsx

```diff
@@ -262,13 +269,16 @@ export default function CampaignDetailsSettingsTab() {
             {showTooltip && (
```

Comment:
Hey I know this is not part of the PR but what are your thoughts on showing the tooltips only if `shouldShowExistingUserIntroModal` is false? @jdghinson has mentioned that he would prefer not to be able to see the tooltip through the blur of the `exisitingUserIntroModal`.

Reply:
@JoshuaHungDinh Good idea! Resolved in df554d989a50645201ac88e07ba5836660fd06a9